plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/DefaultJdbcMetadata.java

```diff
@@ -478,7 +486,8 @@ public Optional<JoinApplicationResult<ConnectorTableHandle>> applyJoin(
                                                 .addAll(leftReferencedTables)
                                                 .addAll(rightReferencedTables)
                                                 .build())),
-                        nextSyntheticColumnId),
+                        nextSyntheticColumnId,
+                        Optional.empty()),
```

Comment:
It should not be empty. Take authorization from one of the table handles as both have the same.